### PR TITLE
[15.0][ADD] event_min_seat: Minimum seats in events

### PR DESCRIPTION
**Pull request by pedrobaeza**
This module recovers the lost feature for specifying the minimum seats required for an event, and showing such events on red in the list view if the minimum seats are not satisfied.

@Tecnativa TT42014

### CODE REVIEW

File: event_min_seat/views/event_event_views.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<odoo>
+    <record model="ir.ui.view" id="view_event_form">
+        <field name="model">event.event</field>
+        <field name="inherit_id" ref="event.view_event_form" />
+        <field name="arch" type="xml">
+            <field name="seats_limited" position="after">
+                <span
+                    attrs="{'invisible': [('seats_limited', '=', False)], 'required': [('seats_limited', '=', True)]}"
+                >from <field name="seats_min" class="oe_inline" /></span>
```
Comment (chienandalu) [inline event_min_seat/views/event_event_views.xml:9]: image

```suggestion
                >from <field name="seats_min" class="oe_inline" />&amp;nbsp;</span>
```

File: event_min_seat/views/event_type_views.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<odoo>
+    <record model="ir.ui.view" id="view_event_type_form">
+        <field name="model">event.type</field>
+        <field name="inherit_id" ref="event.view_event_type_form" />
+        <field name="arch" type="xml">
+            <field name="has_seats_limitation" position="after">
+                <span
+                    attrs="{'invisible': [('has_seats_limitation', '=', False)], 'required': [('has_seats_limitation', '=', False)]}"
+                >
+                    from <field name="default_registration_min" class="oe_inline" />
```
Comment (chienandalu) [inline event_min_seat/views/event_type_views.xml:10]:
```suggestion
                    from <field name="default_registration_min" class="oe_inline" />&amp;nbsp;
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
Tiny detail

**Review by chienandalu — APPROVED** [review]
:+1:

File: event_min_seat/views/event_event_views.xml (hunk 1 above)
Comment (pedrobaeza) [reply event_min_seat/views/event_event_views.xml:9]: Thanks for the suggestion. I was looking for a way to separate them, but didn't find it.

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**